gpbp/data_src.py

```diff
@@ -148,8 +146,8 @@ def osm_facilities(
     for an area defined by the MultiPolygon geometry
     """
     print(f"Retrieving {tags} for {adm_name} area")
-    gdf = ox.geometries_from_polygon(polygon=geometry, tags=tags)
-    osmids = gdf.index.get_level_values("osmid")
```

Comment:
For some reason this function doesn't use `osmid` anymore, but the rest of the `osmnx` package does, so I just had to change it here.